src/api/app/models/attrib.rb

```diff
@@ -34,7 +34,7 @@ class Attrib < ApplicationRecord
            :validate_issues,
            :validate_allowed_values_for_attrib_type
 
-  after_commit :write_container_attributes, on: [:create, :destroy]
+  after_commit :write_container_attributes, on: [:create, :destroy, :update]
```

Comment:
this is still called explicitly in webui - and I had a reason back then, but I don't remember

Reply:
probably because it wasn't being called here?